discussion_of_helm_probe_and_resource.md

```diff
@@ -0,0 +1,35 @@
+Let's use this file for a discussion of how to set up the Helm probe and resource in common helm chart, e.g. tgi, tei, etc.
+
+Below is the proposal from @eero-t :
+
+[1] What's IMHO needed to fix things for probe timings:
+
+- Common components' value files include different probe timing sections for CPU and for accelerators
+
+- Their deployment templates select one based on .Values.accelDevice value (empty for CPU)
+
+- All <device>-values.yaml files set appropriate <subchart>.accelDevice value (not just ChatQnA)
+
+- GMC device variant manifests are generated for all relevant components, not just TGI
+
+(I don't think probe timings would need to be fine-tuned based on which model is used.)
```

Comment:
The startup time varies a lot, for different CPU models combined with different AI models/Size. And the tgi version matters too per my limited observation.
However, for a dedicated AIExample, we'll have default model/tgi, the default probe timing can be set accordingly and remind in the docs for the possibility of tuning.

Reply:
> The startup time varies a lot, for different CPU models combined with different AI models/Size. And the tgi version matters too per my limited observation.

AFAIK there's a magnitude of performance difference between accelerator devices (real ones, not e.g. iGPUs and small NPUs) and CPUs for inferencing.

@yongfengdu Are you saying that if you use _largest_ model (fitting to single device) listed in OPEA docs on Gaudi TGI, and _smallest_ model listed in OPEA docs on on Xeon TGI, latter starts faster because there's such a large difference between those models?